File: dnacount/dnacount.c
#include<stdio.h>
#include<stdlib.h>

int main() {
  int count[256] = { 0 };
  int k;
  FILE *fp = fopen("./rosalind_dna.txt", "r");
  int c;
  while ((c = fgetc(fp)) != EOF) {
    if (c == '\n') continue;
    count[c]+=1;
  }
  for(k=0; k<256; k++) {
    if (count[k] > 0)
      printf("Nu: %c: %d\n", k, count[k]);
  }
  fclose(fp);
  return 0;
}

File: dnacount/dnacount.py
#!/usr/bin/env python
file = open('./rosalind_dna.txt', 'r')
string = file.read().strip()
freq = {'A': 0, 'C': 0, 'G': 0, 'T': 0}
for i in string:
    freq[i] += 1

print freq['A'], freq['C'], freq['G'], freq['T']

# infile = open("./rosalind_dna.txt", "r")
# seq = infile.read()

# def qt(s):
#     return s.count("A"), s.count("G"), s.count("C"), s.count("T")

# print(qt(seq))
